src/Elastic.Apm/Report/PayloadSenderV2.cs

```diff
@@ -44,6 +44,8 @@ internal class PayloadSenderV2 : BackendCommComponentBase, IPayloadSender
 		{
 			_logger = logger?.Scoped(ThisClassName + (dbgName == null ? "" : $" (dbgName: `{dbgName}')"));
 
+			_intakeV2EventsAbsoluteUrl = BackendCommUtils.ApmServerEndpoints.BuildIntakeV2EventsAbsoluteUrl(config.ServerUrls.First());
```

Comment:
These changes are not related to central config, right? Is it because you think we'd have the same relative path problem with the events endpoint?